DOC-397 - added note for flag day (#1804)
corda/corda-docs-portal: content/en/platform/corda/4.11/community/network-map.md

1. Advertise the proposed network parameter update to the entire network.
2. Switching the network onto the new parameters - also known as a *flag day*.

{{< note >}}

> When the flag day is run, all nodes (regardless of whether they have accepted or not) will shutdown. The nodes who have previously accepted the update can just be restarted. The nodes who did not accept will need to manually purge their network parameters file before being allowed to restart.

Reviewer: ```suggestion
When a flag day is run, all nodes (regardless of whether they have accepted or not) shut down. The nodes that previously accepted the update can be restarted. The nodes that did not accept must manually purge their network parameters file before restarting.
```
